src/Symfony/Component/Config/Tests/Resource/GlobResourceTest.php

```diff
@@ -165,4 +165,21 @@ public function testIsFreshRecursiveDetectsNewFile()
         touch($dir.'/Resource/TmpGlob');
         $this->assertFalse($resource->isFresh(0));
     }
+
+    public function testBraceFallback()
+    {
+        $dir = \dirname(__DIR__).\DIRECTORY_SEPARATOR.'Fixtures';
+        $resource = new GlobResource($dir, '/*{/*/*.txt,.xml}', true);
+
+        $p = new \ReflectionProperty($resource, 'globBrace');
+        $p->setAccessible(true);
+        $p->setValue($resource, 0);
+
+        $expected = [
+            $dir.'/Exclude/ExcludeToo/AnotheExcludedFile.txt',
```

Comment:
Typo in AnotheR

Reply:
Haha that's the name of the file, so be it :)